docs/src/main/sphinx/security/overview.rst

```diff
@@ -0,0 +1,150 @@
+provides an introduction to different aspects of configuring security for your
+Trino cluster.
+
+Aspects of configuring security
+-------------------------------
+
+The default installation of Trino has no security features enabled. Security
+can be enabled for different parts of the Trino architecture:
+
+* :ref:`security-client`
+* :ref:`security-inside-cluster`
+* :ref:`security-data-sources`
+
+Suggested configuration workflow
+--------------------------------
+
+To configure security for a new Trino cluster, follow this best practice
+order of steps. Do not skip or combine steps.
+
+#. **Enable** :doc:`HTTPS and TLS </security/tls>`
+
+   * Work with your security team.
+   * Use a :ref:`load balancer or proxy <https-load-balancer>` to terminate
+     HTTPS, if possible.
+   * Use a globally trusted TLS certificate.
+
+#. **Enable authentication**
+
+   * Start with :doc:`password file authentication <password-file>` to get up
+     and running.
+   * Then configure your preferred authentication provider, such as :doc:`LDAP
+     </security/ldap>`.
+   * Avoid the complexity of Kerberos for client authentication, if possible.
+
+#. **Enable authorization and access control**
+
+   * Start with :doc:`file-based rules <file-system-access-control>`.
+   * Then configure another access control method as required.
+
+Configure one step at a time. Always restart the Trino server after each
+change, and verify the results before proceeding.
+
+.. _security-client:
+
+Securing client access to the cluster
+-------------------------------------
+
+Trino :doc:`clients </client>` include the Trino :doc:`CLI </installation/cli>`,
+the :doc:`Web UI </admin/web-interface>`, the :doc:`JDBC driver
+</installation/jdbc>`, `Python, Go, or other clients
+<https://trino.io/resources.html>`_, and any applications using these tools.
+
+All access to the Trino cluster is managed by the coordinator. Thus, securing
+access to the cluster means securing access to the coordinator.
+
+There are three aspects to consider:
+
+* :ref:`cl-access-encrypt`: protecting the integrity of client to server
+  communication in transit.
+* :ref:`cl-access-auth`: identifying users and user name management.
+* :ref:`cl-access-control`: validating each user's access rights.
+
+.. _cl-access-encrypt:
+
+Encyryption
+^^^^^^^^^^^
+
+The Trino server uses the standard :doc:`HTTPS protocol and TLS encryption
+<tls>`, formerly known as SSL.
+
+.. _cl-access-auth:
+
+Authentication
+^^^^^^^^^^^^^^
+
+Trino supports several authentication providers. When setting up a new cluster,
+start with simple password file authentication before configuring another
+provider.
+
+* :doc:`Password file authentication <password-file>`
+* :doc:`LDAP authentication <ldap>`
+* :doc:`OAuth 2.0 authentication <oauth2>`
+* :doc:`Salesforce authentication <salesforce>`
+* :doc:`Kerberos authentication <server>`
+* :doc:`Kerberos authentication with CLI <cli>`
```

Comment:
remove this item since it is just part of Kerberos and the CLI realistically .. not a separate authentication

Reply:
That leaves an existing file out of the Overview. But on balance, OK.